CHANGES.rst

```diff
@@ -21,11 +20,27 @@ Enhancements
 
 
 
+Bug fixes
+----------
+- Fixing docs build failure on Travis [#215]
+
+
+2.3.3 (2019-02-03)
+==================
+This is a bug-fix release and contains general code quality improvements.
+
+
+Enhancements
+------------
+
+
+
 Bug fixes
 ----------
 - Installation does not require python(3)-config anymore [#209, #211]
 - Better handling of terminal colours for unknown terminals [#209]
-
+- Prevent incorrect calculations with periodic boundaries for large ratios of
+(zmax, Rmax) to Lbox [#210]
```

Comment:
Is this supposed to be in this PR?

Reply:
This was in the release notes but not in the CHANGES.rst  - so I added it in.